screens/CustomerLookupScreen.js

```diff
@@ -17,17 +17,28 @@ export default class CustomerLookupScreen extends React.Component {
       clerkName: '',
       phoneNumber: '',
       errorMsg: '',
-      customerPermission: false
+      customerPermission: false,
+      loading: true
     };
   }
 
   // TODO: this is currently not being used
   // Clears error state and phoneNumber entered so far
   async componentDidMount() {
-    const clerkName = await AsyncStorage.getItem('clerkName');
-    this.setState({ clerkName, phoneNumber: '', errorMsg: null });
+    await this._reset();
+    this.setState({ loading: false });
   }
 
+  _reset = async () => {
+    const clerkName = await AsyncStorage.getItem('clerkName');
+    // Clerk Training: pre-fill customer (Summer Strawberry) phone number
```

Comment:
The pre-filling only happens the first time the component is mounted, and is no longer pre-filling when the clerk finishes checkout and proceeds to next customer. Not sure if this is the intended behaviour - flagging in case it isn't!